test/travis/firebase_init.cc.snip

```diff
@@ -0,0 +1,28 @@
+// Set these to run example.
```

Comment:
Maybe instead you could check for region.

- between `// Set these to run example.` and empty line.
- between `void setup()` and empty line.

that way you can delimit the placeholder value from the sketch specifics, rather than having multiple `.snip`